pkgs/applications/science/electronics/fparser/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fparser";
+  version = "4.5.1";
+
+  src = fetchFromGitHub {
+    owner = "thliebig";
+    repo = "fparser";
+    rev = "a59e1f51e32096bfe2a0a2640d5dffc7ae6ba37b";
+    sha256 = "0wayml1mlyi922gp6am3fsidhzsilziksdn5kbnpcln01h8555ad";
+  };
+
+  buildInputs = [ cmake ];
```

important for cross compilation
```suggestion
  nativeBuildInputs = [ cmake ];
```